server/v2/controllers/userController.js

```diff
@@ -49,6 +49,46 @@ class UserController {
             });
         }
     }
+
+    static async signIn(req, res) {
+        try {
```

Comment:
Remember the same approach you have used to avoid repetitive ```try, catch ``` in your tests. there is also a way you can use it in your routes.  you can remove try-catch in your controllers create a separate function to handle it. that function might return a callback function which takes request, and response but in that case, it will be called in routes. feel free to reach out to me for more clarifications.